### ISSUE TITLE
Password reset emails are being sent to inactive users

### ISSUE THREAD

**Issue by deegeenl** [pennersr/django-allauth#2414]
Users that are marked as inactive should not be able to reset their passwords. This test shows that the `ResetPasswordForm` lets users reset their password even if they are inactive:
```
from allauth.account.forms import ResetPasswordForm
from django.test.client import RequestFactory

User = get_user_model()


class TestResetPasswordForm(TestCase):

    global_request = RequestFactory().get('/')

    def test_user_email_not_sent_inactive_user(self):
        User.objects.create_user('mike123', 'mike@ixample.org', 'test123', is_active=False)
        data = {'email': 'mike@ixample.org'}
        form = ResetPasswordForm(data)
        self.assertTrue(form.is_valid())
        form.save(self.global_request)
        self.assertEqual(len(mail.outbox), 0)
```

As in the Django `PasswordResetForm`, the query for users based on the email address should include `is_active=True`.

**Comment by pennersr** [comment]
Closed via 845aa573

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/2414
